code/game/objects/items/weapons/his_grace.dm

```diff
@@ -125,7 +125,29 @@
 	adjust_bloodthirst(1)
 	force_bonus = HIS_GRACE_FORCE_BONUS * LAZYLEN(contents)
 	playsound(user, 'sound/effects/pope_entry.ogg', 100)
-	icon_state = "his_grace_awakened"
+	DoAnimation()
+
+/obj/item/weapon/his_grace/proc/DoAnimation(start)
+    var/static/list/timings = list(
+    2,
+    1,    
+    2,
+    3)
```

Comment:
I still don't follow, won't the list be recreated every time the proc is called?